File: js/climbing-stairs/climbing-stairs.js
function climbStairs(n){
    let secondLast = 0
    let last = 1
    let temp = 0
    for (let i=1 ; i<=n ; i++){
        temp = secondLast + last
        secondLast = last
        last = temp
    }
    return last
}

let n =4
console.log("answer to the first example is:",climbStairs(n))

File: go/climbing-stairs/climbing-stairs.go
package main

import(
	"fmt"

)

func climbStairs(n int) int {
	secondLast, last := 0, 1
    for i:=1; i<=n; i++ {
        secondLast, last = last, secondLast + last
    }
    return last
}

func main(){
	n := 2
	fmt.Println("answer to the first example is:",climbStairs(n))
}